# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1358916449 18000
#      Tue Jan 22 23:47:29 2013 -0500
# Node ID bbce6de5c0a513f769dfaace9a5769d22aa5a718
# Parent  a0b313c17a6bb2fd47a47a87eb744538d537d6b7
* octave-main-thread.cc: Include <clocale> for setlocale.

diff --git a/libgui/src/octave-adapter/octave-main-thread.cc b/libgui/src/octave-adapter/octave-main-thread.cc
--- a/libgui/src/octave-adapter/octave-main-thread.cc
+++ b/libgui/src/octave-adapter/octave-main-thread.cc
@@ -19,16 +19,17 @@ along with Octave; see the file COPYING.
 <http://www.gnu.org/licenses/>.
 
 */
 
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
+#include <clocale>
 #include <string>
 
 #include "octave.h"
 
 #include "octave-main-thread.h"
 #include "octave-link.h"
 
 octave_main_thread::octave_main_thread () : QThread ()
